README.md

```diff
@@ -1,12 +1,58 @@
-# Webpack Boilerplate
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
->  A simple yet powerful webpack boilerplate, which you can later use as a starting point in your projects.
+# ToDo App
 
+>  A basic  web app that allows you to add todo task.
 
-## Features
 
-- Webpack setup 
-- HtmlWebpackPlugin
-- Loading CSS setup
-- Webpack-dev-server
+## Built With
 
+- HTML
+- CSS
+- Javascript (ES6)
+- Webpack
+- Linters(Lighthouse, Webhint, Stylelint, Eslint)
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+install git, nodejs, any code editor in your local machine
+
+### Setup
+clone the repo or download from https://github.com/Fourteen98/To-Do-App
+
+### Install dependencies
+
+'npm install'
+
+
+### Build the app
+
+`npm run build`
+
+
+### Run the app
+
+`npm start `
+This automatically start app on localhost:3000
+
+### Deployment
+
+To run the project run the "index.html" file in your browser.
+
+
+## Authors
+
+👤 **Muhyideen Elias**
+
+- GitHub: [@githubhandle](https://github.com/fourteen98)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONANL] Please update your Readme, currently some headings are missing such as  
```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse
```


- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.